client/Components/Costs.jsx

```diff
@@ -7,16 +7,15 @@ const Costs = (props) => {
   const {
     price, service, tax, cleaning,
   } = data;
-  console.log(props);
   if (duration) {
     return (
       <div className={styles.fee_lines}>
         <span className={styles.left_item}>
           $
           {price}
-          &nbsp;x&nbsp;
+          {' '}x{' '}
```

Comment:
Interesting means of getting the space. Is there a reason you chose to do it this way instead of using a single set of braces? Or to turn these lines into a template literal between one set of braces?

Reply:
A template literal makes more sense.

Robin left a comment that `{' '}` was more standard than using the HTML space character escapes that I was using before.